bagitobjecttransfer/recordtransfer/templates/recordtransfer/transfer_delete.html

```diff
@@ -1,16 +1,22 @@
-{% extends 'recordtransfer/base.html' %}
+{% extends "recordtransfer/base.html" %}
 {% load i18n %}
-
-{% block title %}{% trans "Delete Transfer?" %}{% endblock %}
-
+{% block title %}
+    {% trans "Delete Transfer?" %}
+{% endblock title %}
 {% block content %}
-<div class="title-text">{% block trans %}Do you wish to delete the transfer from {{ last_updated|date:"F j, Y g:ia" }}?{% endblock %}</div>
-<form id="delete-transfer-form" action="" method="post">
-    {% csrf_token %}
-    <input type="hidden" name="transfer_id" value="{{ transfer_id }}" />
-    <p>
-    <input type="submit" name="yes_delete" value="Yes - Delete" />
-    <input type="submit" name="no_cancel" value="No - Cancel" />
-    </p>
-</form>
-{% endblock %}
+    <div class="title-text">{% trans "Confirm In Progress Transfer Deletion" %}</div>
+    <div class="flex-med-width-item margin-top-25px">
+        {% trans "Do you wish to delete the in-progress transfer from" %} {{ transfer.last_updated|date:"F j, Y g:ia" }}{% if transfer.title %} titled "{{ transfer.title }}"{% endif %}?
+    </div>
+    <form id="delete-transfer-form"
+          action="{% url 'recordtransfer:transferdelete' uuid=transfer.uuid %}"
+          method="post">
+        {% csrf_token %}
+        <p>
+            <button type="button"
+                    onclick="window.location.href='{% url 'recordtransfer:userprofile' %}'"
+                    class="red-button">{% trans "Cancel" %}</button>
+            <button type="submit" class="blue-button">{% trans "OK" %}</button>
```

Comment:
Can you add some space between these buttons? They're a bit cramped

![Screenshot 2024-11-14 141538](https://github.com/user-attachments/assets/7882962e-e004-4445-9767-d3a24e7fbc47)

Reply:
Resolved by https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/240/commits/759d910ee2dfdd772ea53546396ff47f443ac357